src/components/PositionedOverlay/tests/PositionedOverlay.test.tsx

```diff
@@ -131,6 +132,51 @@ describe('<PositionedOverlay />', () => {
       );
     });
   });
+
+  describe('preferInputActivator', () => {
+    afterEach(() => {
+      jest.clearAllMocks();
+    });
+
+    it('uses the input to calculate its dimensions when true', () => {
+      const getRectForNodeSpy = jest.spyOn(geometry, 'getRectForNode');
+
+      const activator = document.createElement('div');
+      const input = document.createElement('input');
+      activator.appendChild(input);
+
+      mountWithAppProvider(
+        <PositionedOverlay
+          {...mockProps}
+          preferInputActivator
+          activator={activator}
+        />,
+      );
+
+      expect(
+        getRectForNodeSpy.mock.calls.some(([node]) => node === input),
```

Comment:
Unfortunately, we can't check for the correct `Rect` being passed in the render prop because jsdom will always return `0` for all values. So we thought this is the second best way of assuring that the input has been used.

Reply:
Also `getRectForNode` is called multiple times for different elements so we could not just use `toHaveBeenCalled`

Reply:
Makes sense. Maybe @AndrewMusgrave will have opinion about this.